src/schema/metadata/SoftwareFilters.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: SoftwareFilters
+description: |
+  [Object](https://www.json.org/json-en.html)
+  of temporal software filters applied, or `"n/a"` if the data is
+  not available.
+  Each key:value pair in the JSON object is a name of the filter and an object
+  in which its parameters are defined as key:value pairs
+  (for example, `{"SSS": {"frame": "head", "badlimit": 7},
+  "SpatialCompensation":
+  {"GradientOrder": "Order of the gradient compensation"}}`).
```

Comment:
the examples for SoftwareFilters are a bit specific to each modality. An EEG person without MEG experience wouldn't know what to do with the example here.

I suggest to either use a generic filter example like about an anti-aliasing filter (which everybody recording digitized timeseries must know about), or to make the examples specified to each modality outside of the schema (as "appendage", similar to requirement level).

Generic example would be:

```
{"Anti-aliasing filter": {"half-amplitude cutoff (Hz)": 500, "Roll-off": "6dB/Octave"}}
```

Reply:
Done in 6273e3d. Thanks for the suggestion!